src/main/java/ru/taximaxim/pgsqlblocks/common/ui/TMTreeViewer.java

```diff
@@ -89,15 +90,16 @@ private void selectSortColumn(TreeColumn column, int columnIndex) {
     }
 
     private void createColumns() {
-        for (int i = 0; i < dataSource.numberOfColumns(); i++) {
+        for (Object o : dataSource.getColumns()) {
```

Comment:
// fix dataSource such that columns list type is not erased :)
        // Hint: understand why generics and why it is sometimes untyped